.travis.yml

```diff
@@ -41,6 +41,8 @@ script:
       pip install --user virtualenv
       export PATH=$PATH:$HOME/.local/bin
       ./go py_prep_for_install_release //py:phantomjs_test:run
+      pip install tox
```

Comment:
Yeah, I will give that a try when I have time to return to this patch. Thanks for the tip! 👍